src/tasks_downstream.py

```diff
@@ -764,6 +764,56 @@ def down(c, purge=False):
         c.run(cmd, pty=True)
 
 
+@task(
+    help={
+        "command": "Command to run in the container. "
+        "Options: ['shell', 'logs', 'bash', 'upgradelog', 'pgactivity']",
+    }
+)
+def kube(c, command, namespace="none"):
+    """Run a kubectl command in the provided namespace."""
+
+    yaml_exists = os.path.exists(PROJECT_ROOT / "project.yaml")
```

Comment:
```suggestion
    yaml_exists = os.path.exists(PROJECT_ROOT / ".glo.yaml")
```

I'd avoid `project.yaml` because it's a bit vague and also I'm pretty sure 99 billion other tools will look for it, pyproject.toml is a prime example.

Reply:
Fair